src/web/views/macros/htbhf-pagination.njk

```diff
@@ -0,0 +1,54 @@
+{#
+  htbhfPagination()
+  Next and previous pagination links for pages in a sequence
+
+  params:
+  next: {
+    title: string,
+    path: string
+  }
+  previous: {
+    title: string,
+    path: string
+  }
+#}
```

Comment:
Can we get rid of this commented out section and the one below?

Reply:
These describe the parameters for the macros. You don't want these?